### ISSUE TITLE
Wish: Add shortcuts for starting existing tasks

### ISSUE THREAD

**Issue by krf** [KDAB/Charm#217]
The Charm main window can have up to six tasks in the list, which have been used within this week already.

Introduce shortcuts such as Alt+1, Alt+2, ..., Alt+6 for starting each individual task right away. Useful for quickly switching between them.

**Comment by dfaure-kdab** [comment]
Not sure where you see a limit at 6 tasks (if that's what you meant), I have 10+ tasks in the time tracker (and its combobox).

The problem with numbered shortcuts is that they'll change every week, or even every time you start a new task that inserts itself into the list. Might be better to allow configuring custom shortcuts, so you can set Ctrl+Alt+I for "IPC", Ctrl+Alt+S for "Soap" or whatever, but based on project names (more stable)?

**Comment by frankosterfeld** [comment]
> On 19 Aug 2015, at 11:37, David Faure <notifications@github.com> wrote:
> 
> Not sure where you see a limit at 6 tasks (if that's what you meant), I have 10+ tasks in the time tracker (and its combobox).
> 
> The problem with numbered shortcuts is that they'll change every week, or even every time you start a new task that inserts itself into the list. Might be better to allow configuring custom shortcuts, so you can set Ctrl+Alt+I for "IPC", Ctrl+Alt+S for "Soap" or whatever, but based on project names (more stable)?
> 

An alternative would be to improve the task selector (currently coming up on “T”) and make it easier to select (recent) tasks and allow starting them with more quick and simple keyboard interaction.

-- 
Frank Osterfeld | frank.osterfeld@kdab.com | Senior Software Engineer
KDAB (Deutschland) GmbH&Co KG, a KDAB Group company
Tel: +49-30-521325470
KDAB - The Qt Experts

**Comment by krf** [comment]
@dfaure-kdab: Yes, I have high Font DPI settings, thus I see less than 10 items I guess.

Configuring custom shortcuts is over the top, IMO. There should be an easier way. Also, having Alt+N-style shortcuts doesn't mean you *have* to use them. I think they would be still useful for some.

@frankosterfeld: Sure, I just don't know a good approach...

**Comment by lingo** [comment]
You can already type <kbd>T</kbd> to select a task by typing...
